codegen/src/main/java/software/amazon/awssdk/codegen/docs/PaginationDocs.java

```diff
@@ -103,20 +151,56 @@ private String getSyncCodeSnippets() {
                         .toString();
     }
 
+    private String getAsyncCodeSnippets() {
+        CodeBlock callOperationOnClient = CodeBlock.builder()
+                                                   .addStatement("$T publisher = client.$L(request)",
+                                                                 asyncPaginatedResponseType(),
+                                                                 getPaginatedMethodName())
+                                                   .build();
+
+        return CodeBlock.builder()
+                        .add("\n\n<p>The following are few ways to use the response class:</p>")
+                        .add("1) Using the forEach helper method. This uses @{@link $T} internally",
```

Comment:
Do they need to care what it uses internally?

Reply:
Nope, removed.